apps/music/elements/music-search-box.js

```diff
@@ -200,14 +159,4 @@ function debounce(fn, ms) {
   };
 }
 
-try {
```

Comment:
I can't remember where this originated now. It was something we found while working on Spark/Customizer.